queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name like '%mon';
+SELECT name FROM animals WHERE date_of_birth > '2016-01-01' AND date_of_birth < '2019-01-01';
```

Comment:
- [ ] Please you excluded '2016-01-01' and '2019-01-01
- [ ] I suggest this:  `date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-01-01'` (List the name of all animals born between 2016 and 2019)

Reply:
Ok thanks for the suggestion, I will just use the BETWEEN  clause.